build/dnvm.ps1

```diff
@@ -84,7 +84,7 @@ Set-Variable -Option Constant "CommandFriendlyName" ".NET Version Manager"
 Set-Variable -Option Constant "DefaultUserDirectoryName" ".dnx"
 Set-Variable -Option Constant "OldUserDirectoryNames" @(".kre", ".k")
 Set-Variable -Option Constant "RuntimePackageName" "dnx"
-Set-Variable -Option Constant "DefaultFeed" "https://www.nuget.org/api/v2"
+Set-Variable -Option Constant "DefaultFeed" "https://www.myget.org/F/aspnetvnext/api/v2"
```

Comment:
Sure, that would work too. We need to do that now or let our team know that they need to set the feed. You cannot build release today without that feed set.

Sort of related: DNX_FEED should be an argument, not an env var https://github.com/aspnet/dnvm/issues/354